pypeit/par/pypeitpar.py

```diff
@@ -2484,6 +2495,15 @@ def validate(self):
         """
         Check the parameters are valid for the provided method.
         """
+        
+        if self.data['tell_npca'] < 1 or self.data['tell_npca'] > 10:
+            raise ValueError('Invalid value {:d} for tell_npca '.format(self.data['tell_npca'])+
+                             '(must be between 1 and 10).')
+                             
+        if self.data['teltype'].lower() != 'pca' and self.data['teltype'].lower() != 'grid':
```

Comment:
Following on from the above comment, I think this might need to be replaced with:

`if self.data['teltype'].lower() not in TelluricPar.valid_teltype():`

Reply:
Done.